index.html

```diff
@@ -160,10 +160,24 @@ <h2>
       </ul>
       <p>
         The details of how to fulfill a payment request for a given <a>payment
-        method</a> are handled by <dfn data-lt="payment handler">payment
-        handlers</dfn>. In this specification, these details are left up to the
-        <a>user agent</a>, but future specifications may expand on the
-        processing model in more detail.
+        method</a> is an implementation detail of a <dfn data-lt=
+        "payment handlers">payment handler</dfn>. Additionally, each payment
```

Comment:
Is this "Additionally," or should it be "Concretely,"? The "can make payment" algorithm seems like a concrete realization of the previous sentence; it doesn't seem like some additional unrelated thing.

Reply:
Making this a definition list. There are few more of these to come...